optaweb-vehicle-routing-backend/pom.xml

```diff
@@ -72,6 +72,12 @@
       <artifactId>graphhopper-reader-osm</artifactId>
       <version>0.11.0</version>
     </dependency>
+    <!-- Enable Spring Boot Automatic Restart, see Development Guide in Readme to learn how to use it. -->
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-devtools</artifactId>
```

Comment:
Side question:
Does spring-boot-devtools work well for you?
I've seen reports on stackoverflow that drools-core.jar and spring-boot-devtools don't get along. Or was it with org.reflections?

Reply:
For the reference, I have found these:
* https://stackoverflow.com/questions/52606152/optaplanner-got-java-reflection-error-in-spring-boot
* https://stackoverflow.com/questions/35746370/optaplanners-drools-working-memory-is-empty

I haven't noticed any issues so far. We'll see. Thanks for mentioning this.